exchanges/binance/binance_websocket.go

```diff
@@ -129,7 +140,6 @@ func (b *Binance) KeepAuthKeyAlive() {
 
 // wsReadData receives and passes on websocket messages for processing
 func (b *Binance) wsReadData() {
-	b.Websocket.Wg.Add(1)
```

Comment:
Good pickup, but this is a widescale problem 😆 👀 . Could you fix this across all exchanges?

Reply:
Yessir